python/hsfs/feature_view.py

```diff
@@ -444,10 +452,10 @@ def create_train_test_split(
             version=None,
             test_size=test_size,
             time_split_size=2,
-            train_start=train_start,
-            train_end=train_end,
-            test_start=test_start,
-            test_end=test_end,
+            train_start=util.convert_event_time_to_timestamp(train_start),
```

Comment:
Raw `train_start` without conversion should be good enough because the conversion occurs in the constructor of TD.